### DISCUSSION TITLE
Post your npub and lets connect!

### DISCUSSION THREAD

**Discussion by PeerRich** [NostrGit/NostrGit#79] [General]
Lemme go first: npub18zx8lw3947pghsgzqv2t0x8pe767sscag5djgj5afr755xkqd97qt530pr (also https://no.str.bio/peer)

Let's connect

**Comment by cypherhoodlum** [comment]
npub1u9e887ad8pl49cxgzqkuljxcxy89dtac7jkyuajnukxg6hu2hufqdjdsp7

**Comment by mfts** [comment]
npub1jdn2awkvcwvvnmlq03ke5a73qy0vx4uqv398zsw6d68uqjnz6ahshknu3c

**Comment by melvincarvalho** [comment]
Possibly a better idea to link your github key to nostr in nostr directory, lots of people have done this already and there are proofs too

https://nostr.directory/p/melvincarvalho

**Comment by melvincarvalho** [comment]
but anyway : npub1melv683fw6n2mvhl5h6dhqd8mqfv3wmxnz4qph83ua4dk4006ezsrt5c24

**Comment by PeerRich** [comment]
> Possibly a better idea to link your github key to nostr in nostr directory, lots of people have done this already and there are proofs too
> 
> https://nostr.directory/p/melvincarvalho

oh i didnt know nostr.directory had github, too. 

nice!

**Comment by jamesindeed** [comment]
Just started with Twitter && Nostr! Loving it so far, just getting the hang of things. :) 

npub1md9lq9jktd9vwdqyet4fnxnzf850sgekyyu9zcgfzwnqx9t2qguq248uxm

**Comment by cypherhoodlum** [comment]
tf is twitter, tho?

**Comment by nflatrea** [comment]
**Twitter** https://twitter.com/B35363
**Mastodon** https://infosec.exchange/@b35363
**Nostr** npub1y0gvydcezdjkg6r3gnudhx8ttjzgvsn5sw0um7ucrdrphapwzn5qy68zn4

**Comment by brycedev** [comment]
npub1305vay5kahkcwakmzse8r0sae7yq6dquwfzp29uugwxpdx2auwusmceyuc

**Comment by pierresisson** [comment]
npub17h8lhl4lpzef8cwfql5wl6x46jnhsrkkr6saulquf4ua8xmy4hrq4aqc2z
twitter: https://twitter.com/pierre6dev

**Comment by PastaGringo** [comment]
Loving Nostr ⚡⚡⚡
npub1ky4kxtyg0uxgw8g5p5mmedh8c8s6sqny6zmaaqj44gv4rk0plaus3m4fd2

**Comment by cypherhoodlum** [comment]
We [decided](https://github.com/NostrGit/NostrGit/discussions/95) to not use Discussions anymore. Let's continue the thread at #121  :heart:
